lua/lvim/plugin-loader.lua

```diff
@@ -81,16 +91,14 @@ function plugin_loader.load(configurations)
     return
   end
   local status_ok, _ = xpcall(function()
+    packer.reset()
     packer.startup(function(use)
       for _, plugins in ipairs(configurations) do
         for _, plugin in ipairs(plugins) do
           use(plugin)
         end
       end
     end)
-    -- colorscheme must get called after plugins are loaded or it will break new installs.
-    vim.g.colors_name = lvim.colorscheme
-    vim.cmd("colorscheme " .. lvim.colorscheme)
```

Comment:
This breaks the colorscheme...

Reply:
https://github.com/LunarVim/LunarVim/pull/2392